src/components/Articles.jsx

```diff
@@ -1,29 +1,30 @@
-import React, { useEffect, useState } from 'react'
-import commentIcon from '../assets/comment-icon.png'
 import voteIcon from '../assets/vote-icon.png'
+import commentIcon from '../assets/comment-icon.png'
+import '../styles/Articles.css'
 import axios from 'axios'
+import React, {useEffect, useState } from 'react'
+import { Link, useSearchParams} from 'react-router-dom'
 import Loading from './Loading'
-import { Link, useSearchParams } from 'react-router-dom'
-import '../styles/Articles.css'
-import Header from './Header'
-import Explore from './Explore'
-
 function Articles() {
-  const [articles, setArticles] = useState([])
+  const [searchParams, setSearchParams] = useSearchParams()
   const [isLoading, setIsLoading] = useState(true);
+  const [articles, setArticles] = useState([])
   const [count, setCount] = useState(5)
-  const [ searchParams, setSearchParams ] = useSearchParams()
-  const topicQuery = searchParams.get('topic')
-
-  console.log(topicQuery);
+    const topicQuery = searchParams.get('topic')
+    const sortByQuery = searchParams.get('sort_by')
+    const orderQuery = searchParams.get('order')
+  
   useEffect(() => {
     axios
-      .get(topicQuery ? `https://be-jw-news.onrender.com/api/articles?topic=${topicQuery}` : 'https://be-jw-news.onrender.com/api/articles')
+      .get(topicQuery ? `https://be-jw-news.onrender.com/api/articles?sort_by=${sortByQuery}&order=${orderQuery}&topic=${topicQuery}` : `https://be-jw-news.onrender.com/api/articles?sort_by=${sortByQuery}&order=${orderQuery}`)
```

Comment:
I would investigate the axios params object as this is very useful when solving this particular problem